requirements.txt

```diff
@@ -10,7 +10,7 @@ attrs==21.4.0
     # via pytest
 beautifulsoup4==4.10.0
     # via -r requirements.in
-cachecontrol[filecache]==0.12.11
+cachecontrol[redis]==0.12.11
```

Comment:
I see that this dependency is flapping: 
https://github.com/openshift-assisted/assisted-installer-deployment/pull/264/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R13